app/jobs/create_uphold_cards_job.rb

```diff
@@ -14,10 +14,6 @@ def perform(publisher_id:)
                                               currency_code: default_currency).perform
     end
 
-    if default_currency != "BAT" && publisher.wallet.address.blank?
-      UpholdServices::CardCreationService.new(publisher: publisher, currency_code: "BAT").perform
-    end
-
```

Comment:
This change probably should have happened in #1726.  The idea previously was that we should always create the BAT card when it was possible but not available.  Since we don't use available currencies anymore, we should only create a BAT card when the user requests it.